crates/yul/benches/make_llvm_ir.rs

```diff
@@ -0,0 +1,127 @@
+#![cfg(feature = "bench-llvm-ir")]
+
+use alloy_primitives::*;
+use criterion::{
+    criterion_group, criterion_main,
+    measurement::{Measurement, WallTime},
+    BatchSize, BenchmarkGroup, Criterion,
+};
+use inkwell::context::Context as InkwellContext;
+use revive_integration::cases::Contract;
+use revive_llvm_context::{
+    initialize_llvm, Optimizer, OptimizerSettings, PolkaVMContext, PolkaVMTarget, PolkaVMWriteLLVM,
+};
+use revive_yul::{lexer::Lexer, parser::statement::object::Object as AstObject};
+
+/// Function under test - Lowers the Yul AST Object into LLVM IR.
+fn make_llvm_ir(mut ast: AstObject, mut llvm_context: PolkaVMContext) {
+    ast.declare(&mut llvm_context)
+        .expect("expected LLVM IR generation");
+    ast.into_llvm(&mut llvm_context)
+        .expect("expected LLVM IR generation");
```

Comment:
```suggestion
        .expect("the AST should lower to LLVM IR");
```

Same here, please either a meaningful message or no message.